Tests/IntegrationTests/Media+Edge+EdgeIdentityFunctionalTests.swift

```diff
@@ -15,11 +15,36 @@ import AEPEdge
 import AEPEdgeIdentity
 @testable import AEPEdgeMedia
 import AEPServices
+import AEPTestUtils
 import Foundation
 import XCTest
 
-class EdgeMediaIntegrationTests: FunctionalTestBase {
+class EdgeMediaIntegrationTests: TestBase, AnyCodableAsserts {
+    private let mockNetworkService = MockNetworkService()
     private let sessionStartEdgeEndpoint = "https://edge.adobedc.net/ee/va/v1/sessionStart"
+
+    private let baseEdgeEndpoint = "https://edge.adobedc.net/ee/va/v1/"
+
+    enum MediaPath: String, CaseIterable {
+        case adBreakComplete
```

Comment:
nit: Can we use the internal XDMMediaEventType enums: https://github.com/adobe/aepsdk-edgemedia-ios/blob/main/Sources/xdm/XDMMediaEventType.swift?